en/docs/Learn/ConsumeAPI/Customizations/adding-a-user-signup-workflow.md

```diff
@@ -204,11 +204,18 @@ This section explains how to attach a custom workflow to the user signup operati
         All workflow process services of the EI/BPS run on port 9765 because you changed its default port (9763) with an offset of 2.
 
 3.  Go to the Developer Portal Web interface of API Manager and sign up / register as a new user.
+<html>
+    <body>
+        <div>
+            <img src="../../../../assets/img/Learn/register-now.png" alt="Register now option" width="600"/>
+        </div>
+    </body>
+</html>
     It invokes the signup process and creates a Human Task instance that holds the execution of the BPEL until some action is performed on it.
 
 4.  Note the message that appears if the BPEL is invoked correctly, saying that the request is successfully submitted.
 
-5.  Log in to the Admin Portal ( `https://:9443/admin` ) of API Manager and approve the user signup task. It resumes the BPEL process and completes the signup process.
+5.  Log in to the Admin Portal ( `https://<Server-Host>:9443/admin` ) of API Manager and approve the user signup task listed under **Tasks** > **User Creation**. It resumes the BPEL process and completes the signup process.
```

Comment:
shall we make this a link?

Reply:
shall we reword it and split into points as below,
" Log in to the Admin Portal ( `https://<Server-Host>:9443/admin` ) of API Manager giving the admin username and password."
"Navigate to **Tasks** > **User Creation** and approve the user signup task listed ".
This will resume the BPEL process and complete the signup process

Reply:
Fixed